plugin/evm/message/codec.go

```diff
@@ -22,7 +22,7 @@ var (
 
 func init() {
 	Codec = codec.NewManager(maxMessageSize)
-	c := linearcodec.NewDefault()
```

Comment:
May be worth comparing to how we initialize this in AvalancheGo: https://github.com/ava-labs/avalanchego/blob/7623ffd4be915a5185c9ed5e11fa9be15a6e1f00/vms/proposervm/block/codec.go#L24-L25

Reply:
I think it would be safe to use large maximums here, though I would prefer to make a "smaller" change

Reply:
I think `Codec.Manager` adds `ShortLen` for the codec version?

So, setting them equal may not work: https://github.com/ava-labs/avalanchego/blob/7623ffd4be915a5185c9ed5e11fa9be15a6e1f00/codec/manager.go#L126

Reply:
Other than compatibility reasons, I think we're safe to pass `MaxInt` for all the maximum slice length logic... So I think this is fine. Previously this could have caused 64 MiBs to be allocated when parsing `CodeRequest` with just a few bytes (because the slice would be pre-allocated). However, the codec doesn't do that anymore.

Reply:
@StephenButtolph  This has been limiting the size of the message that gets sent through SendAppGossip as well, is that safe too?

Reply:
I think it will still be limited by the `maxMessageSize`, right?

Reply:
Yes this is safe.